### PR TITLE
User/leober/update sdn express latest changes 11 17

### PR DESCRIPTION
**Pull request by berezleona**

### CODE REVIEW

File: SDNExpress/scripts/SDNExpressModule.psm1 [hunk 1]
```diff
@@ -3212,9 +3434,35 @@ function Test-SDNExpressHealth
         write-sdnexpresslog "$($Server.properties.connections.managementaddresses) status: $($server.properties.configurationstate.status)"
     }
     write-sdnexpresslog "Mux Status:"
-    $muxes = get-networkcontrollerloadbalancermux @DefaultRestParams
-    foreach ($mux in $muxes) {
-        write-sdnexpresslog "$($mux.ResourceId) status: $($mux.properties.configurationstate.status)"
+    [int]$muxRetires = 10
```
Comment (vlappenbusch) [inline SDNExpress/scripts/SDNExpressModule.psm1:3437]: Typo in `muxRetries`

**Review by vlappenbusch — COMMENTED** [review]
Just taking a look at this now - can you leave a comment to share what this is addressing so that we have a log of the changes? I read through the diff and all of the code seems to make sense in isolation, but any context you have would be appreciated!

**Comment by berezleona** [comment]
These are the latest changes that we run in SDDC lab; they improve stability and functionality and these changes are tested by running in the lab for more than a month

**Review by vlappenbusch — APPROVED** [review]
Just got confirmation that I can approve this - I'll approve now but if you're able to fix that typo in the module that would be great!

**Pull request merged**